release_dashboard/lib/widgets/engine_cherrypicks_substeps.dart

```diff
@@ -62,39 +76,89 @@ class ConductorSubstepsState extends State<EngineCherrypicksSubsteps> {
 
   @override
   Widget build(BuildContext context) {
+    List<String> engineCherrypicksInConflict = <String>[];
+    if (context.watch<StatusState>().releaseStatus != null &&
+        context.watch<StatusState>().releaseStatus?['Engine Cherrypicks'] != null) {
```

Comment:
`releaseStatus`'s keys are hard-coded strings. I will make another PR to convert them to enums.

Reply:
Adding a TODO and assigning an issue is a great way to track this work

Reply:
Sounds good, created one [here](https://github.com/flutter/flutter/issues/93730)

Reply:
Also add it to the code so others can track what future features / improvements are planned

Reply:
Oops, my bad, forgot about it. I added the todo here: https://github.com/Yugue/cocoon/blob/0ce9c7b1d0a902ba9fe70bf2649c58a59baffc6e/release_dashboard/lib/state/status_state.dart#L45